### DISCUSSION TITLE
Are there any plans to implement client SDKs in other languages? Like java

### DISCUSSION THREAD

**Discussion by suoc** [weyoss/redis-smq#129] [Q&A] [answered]
Are there any plans to implement client SDKs in other languages? Like java

**Comment by weyoss** [comment, marked as answer]
Implementing RedisSMQ in other languages is planned but it is not a top priority. Currently my focus is mainly dedicated to improving and making better this message queue. 

Golang is preferred over Java as it is optimized for performance and simplicity.

**Comment by weyoss** [comment]
Closing as resolved.
